Projects/Claims/Sources/Views/submitClaim/Checkout/SubmitClaimCheckoutNoRepairScreen.swift

```diff
@@ -64,19 +64,20 @@ public struct SubmitClaimCheckoutNoRepairScreen: View {
         }
     }
 
-    @ViewBuilder func displayPriceFields(checkoutStep: FlowClaimSingleItemCheckoutStepModel?) -> some View {
+    @ViewBuilder
+    func displayPriceFields(checkoutStep: FlowClaimSingleItemCheckoutStepModel?) -> some View {
         displayField(withTitle: L10n.Claims.Payout.Purchase.price, andFor: checkoutStep?.price)
         Divider()
-        displayField(withTitle: L10n.Claims.Payout.Age.deduction, andFor: checkoutStep?.depreciation)
+        displayField(withTitle: L10n.Claims.Payout.Age.deduction, andFor: checkoutStep?.depreciation, prefix: "-")
```

Comment:
We should add a space after the minus sign here, as it is in the figma design https://www.figma.com/file/klLSfgShhdy1n37O2nXf3d/Claims-1.1?node-id=1800-40955&t=j8mqmnBRQgZ7X0uN-4

Reply:
Added